python/ray/tests/test_dashboard_profiler.py

```diff
@@ -23,7 +23,7 @@
 )
 @pytest.mark.parametrize("native", ["0", "1"])
 def test_profiler_endpoints(ray_start_with_dashboard, native):
-    # Sanity check py-spy is installed.
+    # Sanity check py-spy and are installed.
     subprocess.check_call(["py-spy", "--version"])
```

Comment:
This looks like a typo– did you mean to check that `py-spy` and `memray` are installed? That also requires adding a `subprocess.check_call()` call for `memray`.